pkg/restapi/model/model.go

```diff
@@ -30,3 +30,10 @@ type DataProfile struct { // nolint: unused
 	Created                 *time.Time `json:"created,omitempty"`
 	DIDPrivateKey           string     `json:"didPrivateKey,omitempty"`
 }
+
+// LogSpec represents a logging level specification.
+// Format: ModuleName1=Level1:ModuleName2=Level2:ModuleNameN=LevelN:AllOtherModuleDefaultLevel
+// Example: restapi=debug:edvrest=info:info
+type LogSpec struct {
```

Comment:
Can't we move this inside `logspec` pkg, if it's not reused across multiple pkgs ?

Reply:
You're right - done.